CorsixTH/Lua/languages/russian.lua

```diff
@@ -2610,7 +2610,7 @@ humanoid_name_ends = {
   [26] = "атич",
 }
 vip_names = {
-  [1] = "Мэр Большого Трумптона",
+  [1] = "Мэр Нижнего Магнитошахтинска",
```

Comment:
Matroftt did you want me to rebase this PR for you after #2637 merges (afaik all PRs are currently held until release now)? Otherwise this string change will clash.

Reply:
sure

Reply:
I've updated my PR with Matroftt's suggested VIP strings, so there shouldn't be any clash once it's been merged.